src/Chapter14/Listing14.18.CSharpConceptualEquivalentOfEventCILCode.cs

```diff
@@ -33,19 +35,22 @@ public Thermostat(EventHandler<TemperatureArgs>? onTemperatureChange)
             System.Delegate.Remove(_OnTemperatureChange, handler);
         }
 
-        //public event EventHandler<TemperatureArgs> OnTemperatureChange
-        //{
-        //    //Would cause a compiler error
-        //    add
-        //    {
-        //        add_OnTemperatureChange(value);
-        //    }
-        //    //Would cause a compiler error
-        //    remove
-        //    {
-        //        remove_OnTemperatureChange(value);
-        //    }
-        //}
+        #if ConceptulEquivalendCode
```

Comment:
```suggestion
        #if ConceptualEquivalendCode
```
I think this is mispelled

Reply:
I think "Equivalend" is also a misspelling